testutil/sample/observer.go

```diff
@@ -119,6 +120,30 @@ func ChainParamsList() (cpl types.ChainParamsList) {
 	return
 }
 
+// TSSRandom returns a random TSS,it uses the randomness provided as a parameter
+func TSSRandom(r *rand.Rand) types.TSS {
+	pubKey := PubKey(r)
+	spk, err := cosmos.Bech32ifyPubKey(cosmos.Bech32PubKeyTypeAccPub, pubKey)
+	if err != nil {
+		panic(err)
+	}
+	pk, err := zetacrypto.NewPubKey(spk)
+	if err != nil {
+		panic(err)
+	}
+	pubkeyString := pk.String()
+	return types.TSS{
+		TssPubkey:           pubkeyString,
+		TssParticipantList:  []string{},
+		OperatorAddressList: []string{},
+		FinalizedZetaHeight: r.Int63(),
+		KeyGenZetaHeight:    r.Int63(),
+	}
+
+}
+
+// TODO rename to TSS
```

Comment:
```suggestion
// TODO: rename to TSS
```
Why not doing it here? I assume this is a minimal change